src/styles.css

```diff
@@ -0,0 +1,4 @@
+@import "./page.css";
+@import "./card.css";
+@import "./reset.css";
```

Comment:
Import the 'reset.css' file before other style files to ensure default styles are reset first.

Reply:
Not resolved. Just sort your imports.

Reply:
still not fixed 
<img width="784" alt="image" src="https://user-images.githubusercontent.com/53611910/236622349-46bb44f7-d150-4c92-b736-b849d561f986.png">

Reply:
sorry, in the first commit I push file stiles.css but after I renamed this file on style.css and it has sorted import, there is in commit 'fix all comments'.